services/movie.py

```diff
@@ -0,0 +1,38 @@
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> Movie.objects:
+    queryset = Movie.objects.all()
+
+    if genres_ids is not None:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+
+    if actors_ids is not None:
+        queryset = queryset.filter(actors__id__in=actors_ids)
```

Comment:
You don't need "is not None", checking would work without it

Reply:
idk why i wrote it. just forget you saw it, pls